tests/foreman/ui/test_subnet.py

```diff
@@ -57,19 +62,24 @@ def test_create_subnet_with_long_strings(self, test_data):
         @Assert: Subnet is created with 255 chars
 
         """
-        bug_id = test_data.pop('bz-bug', None)
-        if bug_id is not None and bz_bug_is_open(bug_id):
-            self.skipTest('Bugzilla bug {0} is open.'.format(bug_id))
-
         with Session(self.browser) as session:
-            make_subnet(
-                session,
-                subnet_name=test_data['name'],
-                subnet_network=gen_ipaddr(ip3=True),
-                subnet_mask=gen_netmask(),
-            )
-            self.assertIsNotNone(
-                self.subnet.search_subnet(subnet_name=test_data['name']))
+            for test_data in valid_long_names():
+                with self.subTest(test_data):
+                    bug_id = test_data.pop('bz-bug', None)
```

Comment:
This is not new code and was an existing logic already.  html data is affected by bz 1180066 and it will be skipped inside the test.  Other data will run as usual.